text/0006-architecture-review-process.md

```diff
@@ -40,15 +40,16 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
       - how the proposal handles these scenarios
       - what review and discussion of the proposed architecture has already happened (and with whom)
     * A competitive analysis suggesting alternatives, costs, and opportunities
+    * Input by representatives from the affected groups. The team is expected to listen to this input and update the review packet based on the input, but it is likely that not all the questions or comments can be answered before the review.
```

Comment:
I worry that expecting the team to "listen to this input" will rub people the wrong way, and it seems ancillary to what seems to be the main point, which is that the team is *not* required to answer all questions and address all comments before the review. I would rewrite this sentence to focus on that main point, something like:

* Input by representatives from the affected groups. The team may update the review packet based on this input but is not required to answer all questions and address all comments before the review.

Reply:
Fixed in 921a0c87bf7f621c8daa3f54326fbd173e7c8332